Reword motivation for Schnorr multisig spec

Mengerian opened the pull request.

jasonbcox commented on spec/2019-11-15-schnorrmultisig.md:
```diff
@@ -21,7 +21,7 @@ In [the last upgrade](2019-05-15-upgrade.md), we added Schnorr support to OP_CHE
 Although we could have added support to OP_CHECKMULTISIG as well (which would have been overall simpler), this would conflict with the desire to do batch verification in future: Currently with OP_CHECKMULTISIG validation, it is needed to check a signature against multiple public keys in order to find a possible match. In Schnorr batch verification however, it is required to know ahead of time, wh ...
 
-Schnorr aggregated signatures (with OP_CHECKSIG) are one way to do multisignatures, but they are technically limited and thus far from being a drop-in replacement for the familiar Bitcoin multisig. Besides that, it is also desirable that any existing coin can be spent using Schnorr signatures, and there are numerous OP_CHECKMULTISIG-based wallets and coins in existence that we want to be able to  ...
+Schnorr aggregated signatures (with OP_CHECKSIG) are one way to do multisignatures, but they have different technical properties than the familiar Bitcoin multisig, and thus are far from being a drop-in replacement for it. Besides that, it is also desirable that any existing coin can be spent using Schnorr signatures, and there are numerous OP_CHECKMULTISIG-based wallets and coins in existence th ...
```
Are there any good, short docs that we can give reference to for those interested in the differences? The only one I could find was this, but it's very long and contains other content: https://bitcoincore.org/logs/2016-05-zurich-meeting-notes.html

Mengerian replied:
I'm not aware of anything easily accessible.

Mengerian committed:
Reword motivation for Schnorr multisig spec

Reword part about "technical limitations" of Schnorr aggregated signatures, based on @jasonbcox's comments in Telegram.
